packages/web/src/scss/tools/__tests__/_utilities.test.scss

```diff
@@ -1,6 +1,37 @@
 @use 'true' as test;
 @use '../utilities';
 
+@include test.describe('generate-token-map function') {
+    @include test.it('should return accent tokens filtered by scope for background colors') {
+        @include test.assert-equal(
+            utilities.generate-token-map(
+                'accent-colors',
+                'accent-',
+                'background',
+                (background-basic, background-subtle)
+            ),
+            (
```

Comment:
I'm not sure if we shouldn't introduce some kind of mocked data for Sass tests. We must overwrite the expected value once the accent colors are extended with another variant. Is that desired or not?

Reply:
I am ok with both.

Generally, tests shouldn't depend on something they don't test or can even influence, so we probably should mock the data. However, I still consider SASS unit tests as something solely for us, and I am ok with modifying test results when new accents are introduced. 

So, to answer your question, yes, it would be nice to have a data mock, but it might easily become outside the scope of this issue.